lib/chutney/TorNet.py

```diff
@@ -2467,4 +2452,4 @@ def main():
     return 0
 
 if __name__ == '__main__':
-    sys.exit(main())
+    sys.exit(main())
```

Comment:
Please don't delete this newline.

You might find it easier to do this work with a better editor.